lib/components/user/nav-login-button.tsx

```diff
@@ -75,51 +76,51 @@ const NavLoginButton = ({
               />
             </span>
           }
-          pullRight
         >
-          <li className="header">{displayedName}</li>
-
-          {links &&
-            links.map((link, i) => {
-              if (link.url.startsWith('http')) {
+          <h4>{displayedName}</h4>
```

Comment:
I know this was in response to my comment, but maybe we should make this a `<p>` instead? Since this `h4` is in the nav bar, it's showing up semantically before the `<h1>` which is going to be confusing for screenreaders. I think a `p` should be fine in this context!